src/serialization/serialize-assets.js

```diff
@@ -16,10 +16,18 @@ const serializeAssets = function (runtime, assetType, optTargetId) {
         const currAssets = currTarget.sprite[assetType];
         for (let j = 0; j < currAssets.length; j++) {
             const currAsset = currAssets[j];
-            const asset = currAsset.asset;
+            let asset = currAsset.asset;
+            if (currAsset.broken) {
+                if (currAsset.broken.asset) {
+                    asset = currAsset.broken.asset;
+                } else {
+                    continue;
+                }
+            }
```

Comment:
How about this as a simpler version? It also covers the case where `currAsset.broken` and `currAsset.asset` are both falsy. I think with your new code that would never happen but if it does, this suggestion would just skip the asset instead of breaking serialization altogether.
```suggestion
            const asset = currAsset.broken ? curAsset.broken.asset : currAsset.asset;
            if (!asset) {
                continue;
            }
```

Reply:
Yes I think that makes sense! Thanks!